src/components/SQForm/SQFormDateTimePicker.tsx

```diff
@@ -1,19 +1,37 @@
 import React from 'react';
-import PropTypes from 'prop-types';
+import moment from '@material-ui/pickers/adapter/moment';
 import Grid from '@material-ui/core/Grid';
 import TextField from '@material-ui/core/TextField';
-import {DateTimePicker} from '@material-ui/pickers';
+import {DateTimePicker, BaseDateTimePickerProps} from '@material-ui/pickers';
 import {ClickAwayListener, makeStyles} from '@material-ui/core';
 import {useForm} from './useForm';
+import BaseFieldProps from '../../types/BaseFieldProps';
+import {ParsableDate} from '@material-ui/pickers/constants/prop-types';
+
+interface SQFormDateTimePickerProps extends BaseFieldProps {
+  /** Disabled property to disable the input if true */
+  isDisabled?: boolean;
+  /** Required property used to highlight input and label if not fulfilled */
+  isRequired?: boolean;
+  /** Placeholder text used inside the input field to provide hints to the user */
+  placeholder?: string;
+  /** Custom onBlur event callback */
+  onBlur?: React.FocusEventHandler<HTMLInputElement>;
+  /** Custom onChange event callback */
+  onChange?: React.ChangeEventHandler;
```

Comment:
This seems to work, but I don't believe it is the correct type.

Reply:
The SQFormDatePicker did it differently and seems to work fine. I am going to align with that.